forge/ee/routes/billing/index.js

```diff
@@ -16,7 +16,17 @@ module.exports = async function (app) {
     function logStripeEvent (/** @type {StripeEvent} */ event, team, subscription, teamId = null, stripeCustomerId = null) {
         const intro = `Stripe ${event.type} event ${event.data.object.id} from ${stripeCustomerId} received for`
         if (team) {
-            app.log.info(`${intro} team '${team.hashid}'`)
+            if (subscription ||
+                event.type === 'checkout.session.completed' ||
```

Comment:
I think this if statement should be other way around to make it clearer what we're checking, currently it feels backwards:

`/^customer.subscription/.test(event.type)`

Reply:
Solved in [6412a0c](https://github.com/FlowFuse/flowfuse/pull/2965/commits/6412a0c1db373d06df61eef67ec6f674db854715) by setting an explicit flag